core/trino-server/pom.xml

```diff
@@ -26,6 +26,2440 @@
         <process-name>${project.artifactId}</process-name>
     </properties>
 
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>io.trino</groupId>
+                <artifactId>trino-array</artifactId>
+                <version>357-SNAPSHOT</version>
```

Comment:
ins't the same written in top level pom?

Reply:
Removed.